2025-02-26 20:51:51,284 - root - INFO - main.py:31 - Démarrage de l'application d'audit financier
2025-02-26 20:51:51,284 - root - INFO - main.py:32 - Mode DEBUG: True
2025-02-26 20:51:51,285 - root - INFO - main.py:35 - Initialisation des ressources...
2025-02-26 20:59:17,755 - root - DEBUG - main.py:61 - Incoming request: GET http://localhost:8000/api/v1/analysis/files
2025-02-26 20:59:17,756 - root - DEBUG - main.py:63 - Headers: {'host': 'localhost:8000', 'connection': 'keep-alive', 'sec-ch-ua-platform': '"Windows"', 'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/133.0.0.0 Safari/537.36', 'accept': 'application/json', 'sec-ch-ua': '"Not(A:Brand";v="99", "Google Chrome";v="133", "Chromium";v="133"', 'sec-ch-ua-mobile': '?0', 'origin': 'http://localhost:3001', 'sec-fetch-site': 'same-site', 'sec-fetch-mode': 'cors', 'sec-fetch-dest': 'empty', 'referer': 'http://localhost:3001/', 'accept-encoding': 'gzip, deflate, br, zstd', 'accept-language': 'fr-FR,fr;q=0.9,en-US;q=0.8,en;q=0.7'}
2025-02-26 20:59:17,760 - backend.models.trained_detector - INFO - trained_detector.py:21 - Initialisation du modèle de détection d'anomalies
2025-02-26 20:59:17,761 - backend.models.trained_detector - INFO - trained_detector.py:319 - TrainedAnomalyDetector initialisé avec succès
2025-02-26 20:59:17,761 - backend.models.anomaly_detector - INFO - anomaly_detector.py:23 - AnomalyDetector initialisé
2025-02-26 20:59:17,761 - backend.services.analysis_service - INFO - analysis_service.py:35 - Dossier créé/vérifié : C:\Users\berli\OneDrive\Bureau\audit_tool_clean\audit_tool\backend\data\files
2025-02-26 20:59:17,762 - backend.services.analysis_service - INFO - analysis_service.py:35 - Dossier créé/vérifié : C:\Users\berli\OneDrive\Bureau\audit_tool_clean\audit_tool\backend\data\jobs
2025-02-26 20:59:17,762 - backend.services.analysis_service - INFO - analysis_service.py:35 - Dossier créé/vérifié : C:\Users\berli\OneDrive\Bureau\audit_tool_clean\audit_tool\backend\data\results
2025-02-26 20:59:17,767 - root - DEBUG - main.py:67 - Response status: 200
2025-02-26 20:59:17,767 - root - DEBUG - main.py:69 - Response headers: {'content-length': '917', 'content-type': 'application/json', 'access-control-allow-origin': '*', 'access-control-expose-headers': '*'}
2025-02-26 20:59:17,769 - root - DEBUG - main.py:61 - Incoming request: GET http://localhost:8000/api/v1/analysis/files
2025-02-26 20:59:17,769 - root - DEBUG - main.py:63 - Headers: {'host': 'localhost:8000', 'connection': 'keep-alive', 'sec-ch-ua-platform': '"Windows"', 'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/133.0.0.0 Safari/537.36', 'accept': 'application/json', 'sec-ch-ua': '"Not(A:Brand";v="99", "Google Chrome";v="133", "Chromium";v="133"', 'sec-ch-ua-mobile': '?0', 'origin': 'http://localhost:3001', 'sec-fetch-site': 'same-site', 'sec-fetch-mode': 'cors', 'sec-fetch-dest': 'empty', 'referer': 'http://localhost:3001/', 'accept-encoding': 'gzip, deflate, br, zstd', 'accept-language': 'fr-FR,fr;q=0.9,en-US;q=0.8,en;q=0.7'}
2025-02-26 20:59:17,771 - root - DEBUG - main.py:67 - Response status: 200
2025-02-26 20:59:17,772 - root - DEBUG - main.py:69 - Response headers: {'content-length': '917', 'content-type': 'application/json', 'access-control-allow-origin': '*', 'access-control-expose-headers': '*'}
2025-02-26 20:59:17,948 - root - INFO - main.py:38 - Fermeture de l'application, nettoyage des ressources...
